Source/Forms/Gen5/Stationary5.cpp

```diff
@@ -104,40 +112,36 @@ void Stationary5::setupModels()
         ui->comboBoxGeneratorLead->addItem(QString::fromStdString(nature));
     }
 
-    ui->filterGenerator->disableControls(Controls::EncounterSlots);
+    ui->filterGenerator->disableControls(Controls::EncounterSlots | Controls::Ability | Controls::Shiny | Controls::Gender
+                                         | Controls::GenderRatio | Controls::Natures);
     ui->filterSearcher->disableControls(Controls::EncounterSlots | Controls::DisableFilter | Controls::UseDelay);
```

Comment:
I can't see the correct filters when I swap to PIDRNG method.

Reply:
Fixed for BW profiles